### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by kabakTech**
[DEMO LINK](https://kabaktech.github.io/layout_product-cards/)
[TEST REPORT LINK](https://kabaktech.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  border-radius: 5px;
+  background-color: $white-color;
+  margin: 0;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
```
Comment (asakevych) [inline src/index.scss:30]: i think font settings are unnecessary for image 😉

File: src/index.scss [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    }
+
+    &__list-item {
+      background-image: url(./images/star.svg);
+      background-position: 50%;
+      background-repeat: no-repeat;
+      width: 16px;
+      height: 16px;
+      list-style: none;
+      margin-right: 4px;
```
Comment (asakevych) [inline src/index.scss:66]: last star shouldn't have margin-right

File: src/index.scss [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+      font-style: normal;
+      font-weight: 400;
+      align-self: center;
+      color: $dark-color;
+    }
+  }
+
+  &__price {
+    margin: 24px 0 0 16px;
+    justify-self: left;
```
Comment (asakevych) [inline src/index.scss:84]: it's not necessary here

File: src/index.scss [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+      font-size: 10px;
+      font-style: normal;
+      font-weight: 400;
+      align-self: center;
+      color: $dark-color;
+    }
+  }
+
+  &__price {
+    margin: 24px 0 0 16px;
```
Comment (asakevych) [inline src/index.scss:83]: its better to make 16px margings for the whole card, so you shouldn't set it for each element inside it

**Review by asakevych — CHANGES_REQUESTED** [review]
Good job!! just some small changes needed

**Review by choeqq — CHANGES_REQUESTED** [review]
Waiting for any updates @kabakTech 👀

**Comment by kabakTech** [comment]
[DEMO LINK](https://kabaktech.github.io/layout_product-cards/)
[TEST REPORT LINK](https://kabaktech.github.io/layout_product-cards/report/html_report/)

File: src/index.scss [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+  &__price-container {
+    display: flex;
+    margin-top: 24px;
+    justify-content: space-between;
+  }
+
+  &__price {
+    display: flex;
+    font-size: 12px;
+    font-style: normal;
```
Comment (JoyCoffeeAddict) [inline src/index.scss:95]: This is default behaviour, you can check it by removing this property in devtools. The same for `stars__review`.

**Review by JoyCoffeeAddict — CHANGES_REQUESTED** [review]
Please, apply previous suggestions.

**Pull request closed without merging**